tests/topotests/all-protocol-startup/test_all_protocol_startup.py

```diff
@@ -480,6 +480,8 @@ def test_ospfv2_interfaces():
             actual = net['r%s' % i].cmd('vtysh -c "show ip ospf interface" 2> /dev/null').rstrip()
             # Mask out Bandwidth portion. They may change..
             actual = re.sub(r"BW [0-9]+ Mbit", "BW XX Mbit", actual)
+	    actual = re.sub(r"ifindex [0-9]", "ifindex X", actual)
```

Comment:
nit: shouldn't we use `[0-9]+` so that this works with ifindexes >= 10? This might be an issue if we create more complex topologies in the future.

Reply:
well only if we change this particular test though right?  In that case this would stop working

Reply:
For some reason I thought `all-protocol-startup` was used by all topotests. Since that's not the case I think we should be fine with your diff. Merging...